src/Logging/Logging.Abstractions/src/EventId.cs

```diff
@@ -3,42 +3,80 @@
 
 namespace Microsoft.Extensions.Logging
 {
+    /// <summary>
+    /// Identifies a logging event.
+    /// </summary>
     public readonly struct EventId
     {
+        /// <summary>
+        /// Implicitly creates an EventId from the given <see cref="int"/>.
+        /// </summary>
+        /// <param name="i">The <see cref="int"/> to convert to an EventId.</param>
         public static implicit operator EventId(int i)
         {
             return new EventId(i);
         }
 
+        /// <summary>
+        /// Checks if two specified <see cref="EventId"/>'s have the same value.
+        /// </summary>
+        /// <param name="left">The first <see cref="EventId"/>.</param>
+        /// <param name="right">The second <see cref="EventId"/>.</param>
+        /// <returns><code>true</code> if the objects are equal.</returns>
         public static bool operator ==(EventId left, EventId right)
         {
             return left.Equals(right);
         }
 
+        /// <summary>
+        /// Checks if two specified <see cref="EventId"/>'s have different values.
+        /// </summary>
+        /// <param name="left">The first <see cref="EventId"/>.</param>
+        /// <param name="right">The second <see cref="EventId"/>.</param>
+        /// <returns><code>true</code> if the objects are not equal.</returns>
         public static bool operator !=(EventId left, EventId right)
         {
             return !left.Equals(right);
         }
 
+        /// <summary>
+        /// Initializes an instance of the <see cref="EventId"/> struct.
+        /// </summary>
+        /// <param name="id">The numeric identifier for this event.</param>
+        /// <param name="name">The name of this event.</param>
         public EventId(int id, string name = null)
         {
             Id = id;
             Name = name;
         }
 
+        /// <summary>
+        /// Gets the numeric identifier for this event.
+        /// </summary>
         public int Id { get; }
+
+        /// <summary>
+        /// Gets the name of this event.
+        /// </summary>
         public string Name { get; }
 
+        /// <inheritdoc />
         public override string ToString()
         {
             return Name ?? Id.ToString();
         }
 
+        /// <summary>
+        /// Indicates whether the current object is equal to another object of the same type.
```

Comment:
Per earlier comments, what does "equal" truly mean here? Definitely not obvious.